### ISSUE TITLE
won't load

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#454]
```
What's the problem?

I have tried to load ScribeFire-Next  and it won't



What browser are you using?

foxfire  7.01



What version of ScribeFire are you running?
Trying to load ScribeFire-Next
```

-----
Original issue reported on code.google.com by `larry_mi...@primus.ca` on 26 Oct 2011 at 2:20

**Comment by GoogleCodeExporter** [comment]
```
What are you clicking on? At what point does something unexpected happen?
```

-----
Original comment by `cfinke` on 4 Nov 2011 at 2:24
* Changed state: **Pending**

**Comment by GoogleCodeExporter** [comment]
```
Issue 456 has been merged into this issue.
```

-----
Original comment by `cfinke` on 4 Nov 2011 at 2:25

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in version 4.2 and above. Waiting on approval from Firefox
```

-----
Original comment by `scribefi...@gmail.com` on 4 Sep 2013 at 9:32
* Changed state: **Fixed**
